string essid
string bssid
uint32 frequency
string state
int8 signal
float32 bitrate
uint32 rx_pkts
uint32 tx_pkts
uint32 rx_bytes
uint32 tx_bytes

